runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsOverlay.java

```diff
@@ -75,18 +83,51 @@ public Dimension render(Graphics2D graphics)
 		if (config.highlightCurrentTile())
 		{
 			final WorldPoint playerPos = client.getLocalPlayer().getWorldLocation();
-			if (playerPos == null)
+
+			if (lastPlayerPosition != null)
 			{
-				return null;
+				if (!playerPos.equals(lastPlayerPosition))
+				{
+					lastTickPlayerMoved = client.getTickCount();
+				}
+				else if (lastTickPlayerMoved + 1 == client.getTickCount())
+				{
+					lastTimePlayerStoppedMoving = System.nanoTime();
+				}
 			}
 
-			final LocalPoint playerPosLocal = LocalPoint.fromWorld(client, playerPos);
-			if (playerPosLocal == null)
+			lastPlayerPosition = playerPos;
+
+			if (lastPlayerPosition != null)
```

Comment:
What does this guard against? Your code currently reads:

```java
WorldPoint playerPos = client.getLocalPlayer().getWorldLocation();
if (lastPlayerPosition != null) {}
lastPlayerPosition = playerPosition;
if (lastPlayerPosition != null)      // this line
```

Can player position actually be null? If so, it probably impacts the code in the first `if` block prior to this one. If not, I don't understand what this check is actually doing.

Reply:
The old version of the plugin has this check. However, looking at other core runelite plugins shows that `client.getLocalPlayer().getWorldLocation()` is not null checked in `render` (e.g. `ImplingsOverlay`). I'll remove this.